aspnet/fundamentals/logging.rst

```diff
@@ -162,7 +182,7 @@ Examining the Application event log in the Windows Event Viewer, the following e
 Configuring Other Providers
```

Comment:
Ok, when we discussed the article we wanted to at least indicate that such providers existed and that the logging implementation for ASP.NET supported plug-ins by third parties. How should we do that if this whole section goes away?